docs/guide/issues-help.txt

```diff
@@ -1,3 +1,59 @@
 =============
 Issues & Help
-=============
\ No newline at end of file
+=============
+
+Our developer community is vibrant and highly engaged, with wide experience using Node.js with MongoDB.
+Often, the quickest way to get support for
+general questions is through the `mongodb-user google group <http://groups.google.com/group/mongodb-user>`_
+or through `stackoverflow <http://stackoverflow.com/questions/tagged/mongodb+nodejs>`_.
+
+Refer to our `support channels <http://www.mongodb.org/about/support>`_ documentation for more information.
+
+Bugs / Feature Requests
+-----------------------
+
+To report a bug or to request a new feature in the Node.js driver,
+please open a case in our issue management tool, JIRA:
+
+
+* `Create an account and login <https://jira.mongodb.org>`_.
+* Navigate to `the NODE project <https://jira.mongodb.org/browse/NODE>`_.
+* Click **Create Issue**. Provide as much information as possible about the
+  issue and the steps to reproduce it.
+
+Bug reports in JIRA for the Node.js driver and the Core Server (i.e. SERVER) project are **public**.
```

Comment:
Here you mentioned "Core Server", do you mean the mongodb server (I think - this repo: https://github.com/mongodb/mongo)? 
If so, can we rephrase this to say:

"Bug reports in JIRA for the Node.js driver and the Server (i.e Core Server) project are public."

The reason for this requested change is because, to my understanding, the mongodb database is typically referred to as "the MongoDB server" rather than "the Core MongoDB server". I don't think it's a big deal which we choose to use for this doc, but the former is probably more clear